quantumcat/circuit/circuit.py

```diff
@@ -380,4 +380,38 @@ def entangle(self, qubit1, qubit2):
 
     def phase_kickback(self, qubit):
         self.x_gate(qubit)
-        self.h_gate(qubit)
\ No newline at end of file
+        self.h_gate(qubit)
+
+    def not_gate(self, qubit):
+        self.x_gate(qubit)
+
+    def or_gate(self, qubits, target_qubit):
```

Comment:
In all functions, "qubits" can be renamed as "control_qubits" in the paraments and elsewhere